x/application/types/genesis.go

```diff
@@ -34,7 +34,7 @@ func (gs GenesisState) Validate() error {
 		applicationIndexMap[index] = struct{}{}
 	}
 
-	// Check that the stake value for the apps is valid
+	// Check that the stake value for the apps is valid and that the delegatee pubkeys are valid
```

Comment:
```suggestion
	// Check that the stake value for the apps is valid and that the delegatee addresses are valid
```
must've missed this one